manifests/install/rabbitmqadmin.pp

```diff
@@ -3,6 +3,14 @@
 
   require '::archive'
 
+  # Newer versions of Ubuntu (16.04) ship with Python 3 only by default.
+  if $rabbitmq::manage_python {
+    ensure_packages('python')
```

Comment:
https://github.com/puppetlabs/puppetlabs-stdlib#ensure_packages states it takes a list of packages so I'm surprised this works.

Reply:
Yeah, I think it does work, but I forced it to list context, which I think is cleaner.

Reply:
I can switch it to managing it for all platforms, seems kind of silly for RHEL / CentOS, where I think it's a hard dependency for tools like yum / anaconda.

Do we know what the package is called on SUSE?